src/sage/knots/free_knotinfo_monoid.py

```diff
@@ -427,10 +434,10 @@ def from_knot(self, knot, unique=True):
                 if unique:
                     return k
                 else:
-                    return[k]  # to be consistent with get_knotinfo
+                    return [k]  # to be consistent with get_knotinfo
```

Comment:
the `else` could be removed and so indentation reduced.

Reply:
ok, fixed